### PR TITLE
Band aid

### PR DESCRIPTION
**Pull request by BenGitsCode**
- Fixes a lot of typos, broken links in CONTRIBUTE.md
 - Also made a small change to the til-template.

### CODE REVIEW

File: CONTRIBUTE.md [hunk 1]
```diff
@@ -38,29 +37,30 @@ Before creating enhancement suggestions, please check [this list](#before-submit
...
 
 ### Your First TIL  Contribution
 
 Unsure where to begin contributing to TIL?
 
-Don't be! No snippet of knowledge is too small. Did you find
+✨ Don't be! No snippet of knowledge is too small! ✨
 
-If you want to read about using TIL or developing packages in TIL, the [TIL Flight Manual](http://flight-manual.TIL.io) is free and available online. You can find the source to the manual in [TIL/flight-manual.TIL.io](https://github.com/TIL/flight-manual.TIL.io).
+Whether its opening an issue pointing out one of my broken links or typos, or just sharing some small code snippet, trick, hack or anything else--don't overthink it! If it feels incomplete we'll ask you to expound upon it. Make it, think about it tomorrow.
```
Comment (laurpaik-zz) [inline CONTRIBUTE.md]:
```diff
-Whether its
+Whether it's
```

This is another sentence that I had to read twice. You have, like, nested lists in it. Stick to one thing per example if possible. Suggestion:
```diff
-Whether its opening an issue pointing out one of my broken links or typos, or just sharing some small code snippet, trick, hack or anything else--don't overthink it!
+Whether it's opening an issue pointing out one of my broken links or just sharing some small code snippet--don't overthink it!
```

File: CONTRIBUTE.md [hunk 2]
```diff
@@ -38,29 +37,30 @@ Before creating enhancement suggestions, please check [this list](#before-submit
...
 
 ### Pull Requests
 
-* Fill in [the required template](PULL_REQUEST_TEMPLATE.md)
+* There is currently no formal pull request flow or template, I'm more concerned with making it easy and comfortable for people to share
```
Comment (laurpaik-zz) [inline CONTRIBUTE.md]:
```diff
-There is currently no formal pull request flow or template, I'm more concerned with making it easy and comfortable for people to share
+There is currently no formal pull request flow or template. I'm more concerned with making it easy and comfortable for people to share
```

**Review by laurpaik-zz — COMMENTED** [review]
Couple grammar things, mostly calm down on how long your sentences are :)

File: CONTRIBUTE.md (hunk 1 above)
Comment (BenGitsCode) [reply CONTRIBUTE.md]: Yeah, I've been letting too much of that sneak past review— or just not reading them out loud. -_-

Thanks for the diff suggestions!

**Review by laurpaik-zz — APPROVED** [review]
👍 great changes!

**Pull request merged**